stablehlo/tests/stablehlo_legalize_to_vhlo.mlir

```diff
@@ -406,6 +406,18 @@ func.func @default_collective_permute(%arg0: tensor<16x8xf32>) -> tensor<16x8xf3
   func.return %0 : tensor<16x8xf32>
 }
 
+// CHECK-LABEL: "default_collective_broadcast"
+func.func @default_collective_broadcast(%arg0: tensor<16x8xf32>) -> tensor<16x8xf32> {
```

Comment:
TP looks good, can we also add a failure to downgrade to 0.15.0 test (fails since this op didn't exist in version 0.15), similar to:

https://github.com/openxla/stablehlo/blob/bfc711d86f653afb5ade6bf8855e1f6ca81b7e1a/stablehlo/tests/vhlo_to_version_downgrade_invalid.0_10_0.mlir